facebook/__init__.py

```diff
@@ -79,14 +81,15 @@ class GraphAPI(object):
     """
 
     def __init__(self, access_token=None, timeout=None, version=None,
-                 proxies=None, session=None):
+                 proxies=None, session=None, page_id=None):
```

Comment:
Why is `page_id` required here?

I think it is really confusing to introduce a parameter with a general name like this and then only have it be used by one specific API endpoint.